src/Core/Domain/Group/ValueObject/GroupId.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Group\ValueObject;
+
+use PrestaShop\PrestaShop\Core\Domain\Group\Exception\GroupConstraintException;
+
+/**
+ * Holds group identification value
+ */
+class GroupId implements GroupIdInterface
```

Comment:
Shop Group ? Group ?

Reply:
idk it just group :smile: 
At first I thought it should be CustomerGroup, but then I saw this
![Screenshot from 2021-09-21 11-58-08](https://user-images.githubusercontent.com/31609858/134142604-2dbb2e43-146a-4b8f-89ef-16e6f4f7a788.png)
So it would be misleading to call it customer group

Reply:
Hoho 😄  so if I was totally mislead, I think it means the name could really use some improvements. Or some phpdoc to help.

Reply:
hmmm, moving Group under Customer would help!
Ill move it to Customer/Group/ValueObject/GroupId